src/compressor/CMakeLists.txt

```
@@ -23,8 +30,12 @@ if(HAVE_BROTLI)
 endif()
 
 add_library(compressor STATIC ${compressor_srcs})
-if(HAVE_QATZIP)
-  target_link_libraries(compressor PRIVATE ${QATZIP_LIBRARIES})
+if(HAVE_QATZIP AND HAVE_QATDRV)
+  target_link_libraries(compressor PRIVATE
+                        QatDrv::qat_s
+                        QatDrv::usdm_drv_s
+                        qatzip::qatzip
```

why are we repeating these linkages here? if any of header file dependencies is "leaked" into compressor library, shall we make "compressor_objs" a static library, and claim the library which include the "leaked" header file as "PUBLIC"? presumably it is `qatzip.h` inclueded by `QatAccel.h`. but it is removed by this change, no?
```suggestion
                        qatzip::qatzip
```